etlman/projects/tests/test_views.py

```diff
@@ -1,21 +1,83 @@
 from http import HTTPStatus
 
 import pytest
-from django.test import Client
+from django.test import Client, TestCase
```

Comment:
If we use `TestCase` here, let's use it in the other file too. Then you can (should) replace any usage of `assert` with its `unittest` equivalent (e.g., `assertEquals()` instead of `assert foo == bar`).

Reply:
Good catch. I prefer pytest so I opted for removing it. I'd love to have a conversation about this because the difference isn't really clear in my mind.

Reply:
As I understand it, you can write your tests either way with pytest. And I while think we **should** keep using pytest to run the tests, for consistency's sake, it's probably best to write them all with `unittest`. This page talks some about how you can run `unittest` test cases with pytest (it just works, but it's still informative to read):

https://docs.pytest.org/en/7.1.x/how-to/unittest.html